package.json

```diff
@@ -47,6 +50,10 @@
     "truncate": "^2.0.1",
     "yarn": "^1.12.1"
   },
+  "resolutions": {
```

Comment:
For the record: `auth0-lock` depends on older version of React. We force it to use react 16 here to not have react version conflicts.